### PR TITLE
fix(challenges): Fix classic modal styling not rendering

### LINKED ISSUE #14215: Classic Modal has Lost its CSS.
Classic modal has lost all styling post #13673 (static nav)

@BerkeleyTrue

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
Move classic modal less files co-locating them in the app file structure

closes #14215

### CODE REVIEW

File: client/less/classic-modal.less [hunk 1]
```diff
@@ -1,34 +0,0 @@
-.challenge-success-modal {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  height: 50vh;
-
-  .modal-header {
```
Comment (Bouncey) [inline client/less/classic-modal.less]: Somewhere along the way I think we lost the background colour to the `.modal-header`, it must have been inherited. 

So here will need to be 
```css
.modal-header {
  background-color: @brand-primary;
  color: #fff;
  margin-bottom: 0;
}
```

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: client/less/classic-modal.less (hunk 1 above)
Comment (BerkeleyTrue) [inline client/less/classic-modal.less]: Wasn't that removed on purpose? I seem to remember a discussion about this

Comment (Bouncey) [inline client/less/classic-modal.less]: Blame on the file says it hasn't been edited since it was merged. I had it mimic the production version

Comment (BerkeleyTrue) [inline client/less/classic-modal.less]: Cool, I'll add the background color in

**Comment by BerkeleyTrue** [comment]
@Bouncey updated

**Comment by Bouncey** [comment]
## LGTM

:+1: 

Thanks for contributing @BerkeleyTrue :wink:

**Pull request merged**